src/main/java/com/akathist/maven/plugins/launch4j/VersionInfo.java

```diff
@@ -137,6 +163,75 @@ private void setLanguage(net.sf.launch4j.config.VersionInfo ret) {
         ret.setLanguage(languageID);
     }
 
+    public void tryFillOutByDefaults(MavenProject project, File outfile) {
```

Comment:
Can this be in the _default scope_? As `public` it can be called out of the package, but this is rather not needed.

Reply:
Great insight! You're right, a `public` scope is not needed here.